core/src/main/resources/reference.conf

```diff
@@ -49,12 +49,6 @@ gcs {
   defaultLocation = "us-central1"
   bucketLogsMaxAge = "180"
   google-api-uri = "https://www.googleapis.com"
-
-  deploymentManager {
```

Comment:
Does this mean there are helm file deletions or firecloud develop deletions that need to happen?

Reply:
yeah, no helmfile, but there's some firecloud-develop deletion that will need to happen